packages/units/src/context.js

```diff
@@ -17,6 +17,6 @@
 /**
  * Internal dependencies
  */
-import { createContext } from '../../design-system';
+import { createContext } from '../../../assets/src/design-system';
```

Comment:
@swissspidy This is an interdependy I don't like but I can live with for now.

Reply:
Yeah, same situation as in `useUnits.js` (https://github.com/google/web-stories-wp/pull/7895/files#r648468991). Totally forgot about this.

It should be possible to fix this soon.

Hope this doesn't cause any issues with tree shaking and bundling in the meantime.